hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestNSSummaryAdmin.java

```diff
@@ -115,7 +115,25 @@ public void testNSSummaryCLIRoot() throws UnsupportedEncodingException {
     String path = "/";
     executeAdminCommands(path);
     // Should throw warning - only buckets can have bucket layout.
-    assertThat(getOutContentString()).contains("[Warning] Namespace CLI is not designed for OBS bucket layout.");
+    assertThat(getOutContentString()).doesNotContain("INVALID_VOLUME_NAME");
+    assertThat(getOutContentString()).doesNotContain(
+        "[Warning] Namespace CLI is not designed for OBS bucket layout.");
+    assertThat(getOutContentString()).contains("Put more files into it to visualize DU");
+    assertThat(getOutContentString()).contains("Put more files into it to visualize file size distribution");
+  }
+
+  /**
+   * Test NSSummaryCLI on volume.
+   */
+  @Test
+  public void testNSSummaryCLIVolume() throws UnsupportedEncodingException {
+    // Running on root path.
+    String path = "/" + volumeName;
+    executeAdminCommands(path);
+    // Should throw warning - only buckets can have bucket layout.
+    assertThat(getOutContentString()).doesNotContain("INVALID_BUCKET_NAME");
+    assertThat(getOutContentString()).doesNotContain(
+        "[Warning] Namespace CLI is not designed for OBS bucket layout.");
```

Comment:
Pls correct the source of this message as NSSummary /du supports all types of buckets now.

Reply:
@devmadhuu If `du` is supported for all buckets should we just avoid doing the validation for bucket layout i.e., `isNotValidBucketOrOBSBucket` in `DiskUsageSubCommand` instead of changing the warning message. Because the warning message is also used by FileSizeDist, QuatUsage and Summary sub commands as well.

Reply:
Yes , we should not validate the layout of bucket now.